### ISSUE TITLE
"Loss is nan" errors

### ISSUE THREAD

**Issue by eminorhan** [facebookresearch/dino#97]
Thanks for making your code and models available! I sometimes get "Loss is nan, stopping training" errors during training when I train a model with DINO (I suspect this is due to some kind of runaway dynamics making logits inf). I'm not sure if there's a consistent pattern to when exactly I get this error, but I was wondering if this is a known, common issue and what I can do to avoid it. Thanks!

**Comment by eminorhan** [comment]
Closing this as this seems to be a duplicate of #58
